raiden-dapp/src/components/notification-panel/NotificationArea.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <v-snackbar
+    v-model="notification.display"
+    :timeout="notification.duration"
+    top
+    centered
+    vertical
+    app
+    rounded
+    max-width="550px"
+    @input="notificationShown(notification.id)"
+  >
+    <v-row no-gutters class="notification-area__notification__content">
+      <v-col
+        v-if="notification.context !== 'none'"
+        cols="auto"
+        class="notification-area__notification__content__icon d-flex flex-column"
+      >
+        <v-img
+          aspect-ratio="1"
+          :class="`notification-area__notification__content__icon--${notification.context}`"
+          contain
+          :src="require(`@/assets/${notification.context}.svg`)"
+          height="48px"
+          width="48px"
+        ></v-img>
+      </v-col>
+      <v-col>
+        <v-row no-gutters>
+          <v-col>
+            <h3 class="text--primary title">{{ notification.title }}</h3>
+          </v-col>
+          <v-col v-if="notificationQueue.length >= 1" cols="auto">
+            <v-badge
+              inline
+              :content="notificationQueue.length"
+              color="info"
+              class="notification-area__notification__content__count"
+            />
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-col>
+            <p
+              class="text--secondary caption notification-area__notification__content__description text-justify"
+            >
+              {{ notification.description }}
+            </p>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+    <template #action="{ attrs }">
```

Comment:
If split/shared, these actions would/could be triggered by the close button instead of an explicit `Dismiss`

Reply:
To be honest, I am not sure I understand what do you mean by that, could you elaborate?

Reply:
The snackbar panel has a `Dismiss` button; the notification panel's card has a close button which instead of dismissing, clears it. If splitting this component into a shared one which would work for both, you'd need to have only one way to do it, possibly with the close button, but then you need to have an event emitted there, instead of hardcoded, because on snackbar, close shouldn't clear the notification from queue, but instead, just hide it.

Reply:
Oh ok! Can we open a followup issue for that?

Reply:
I created the followup issue for that :)